api/v1beta1/verticadb_types.go

```diff
@@ -374,7 +383,7 @@ type CommunalStorage struct {
 
```

Comment:
HadoopConfig should have `xDescriptors="urn:alm:descriptor:io.kubernetes:ConfigMap`